SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2323,14 +2322,16 @@ void USpatialNetDriver::DelayedSendDeleteEntityRequest(Worker_EntityId EntityId,
 	}, Delay, false);
 }
 
-void USpatialNetDriver::HandleStartupOpQueueing(const TArray<Worker_OpList*>& InOpLists)
+void USpatialNetDriver::HandleStartupOpQueueing(const TArray<SpatialGDK::OpList>& InOpLists)
 {
 	if (InOpLists.Num() == 0)
 	{
 		return;
 	}
 
-	QueuedStartupOpLists.Append(InOpLists);
+	// TODO: Discuss
+	//TArray<SpatialGDK::OpList> QueuedStartupOpLists;
```

Comment:
We could make an OpListView type but it's probably easier just to change the QueudStartupOpLists to be an array of OpList*.

Reply:
Oh wait this is meant to own them. In which case we change the argument to not be a const ref and just append the entire array to the queued startup ops. That looks like a mistake in the original code. This is a good example why we shouldn't be using raw pointers for ownership :P.